packages/form-js-editor/src/features/palette/components/Palette.js

```diff
@@ -5,6 +5,10 @@ const types = [
     label: 'Text Field',
     type: 'textfield'
   },
+  {
+    label: 'Text Area',
```

Comment:
Maybe we should update our field labels to [use sentence case](https://camunda.com/brand/writing-style-guide/).

> Text field
> Text area